### PR TITLE
[v3.5] Raven db 10649

### PR DESCRIPTION
**Pull request by maximburyak**
I had trouble writing a test for this scenario, not sure if it's the test or the code.. so we can call it a work in progress

### CODE REVIEW

File: Raven.Database/Raft/Util/RaftHelper.cs [hunk 1]
```diff
@@ -31,7 +31,7 @@ public static NodeConnectionInfo GetLeaderNode(this RaftEngine engine, int waitT
         {            
             if (waitTimeoutInSeconds > 0)
             {
-                if (engine.WaitForLeader(waitTimeoutInSeconds * 1000) == false)
+                if (engine.WaitForLeader(waitTimeoutInSeconds) == false)
```
Comment (ppekrol) [inline Raven.Database/Raft/Util/RaftHelper.cs:34]: @maximburyak are you sure? https://github.com/ravendb/ravendb/blob/v3.5/Rachis/Rachis/RaftEngine.cs#L521

Comment (maximburyak) [reply Raven.Database/Raft/Util/RaftHelper.cs:34]: The parameters name is waitTimeoutInSeconds, I'm sure that from the controllers point of view, I should pass seconds and not multyply by thousands. We'll have to make sure that no code path calls this method with miliseconds though

Comment (ppekrol) [reply Raven.Database/Raft/Util/RaftHelper.cs:34]: `GetLeaderNode` parameter is called `waitTimeoutInSeconds` and yes, it expects the value to be in seconds, but `WaitForLeader` expects the value to be in milliseconds, so there is need of conversion here. Right?

Comment (maximburyak) [reply Raven.Database/Raft/Util/RaftHelper.cs:34]: Yes, I think so. It's good  you  paid attention to that!

**Pull request merged**